net/rdesktop/Portfile

```diff
@@ -33,6 +33,9 @@ depends_lib         port:xorg-libX11 \
                     port:libao \
                     port:libsamplerate
 
+use_autoreconf	yes
+autoreconf.args	-fv
+
```

Comment:
These lines used to be present in the portfile up until 1089d45425. I found that restoring these prevents an error when attempting to compile which wasn't present with 1.8.4. Example log from earlier Azure build: https://paste.z0k.xyz/de6120fab3b9

```
configure: WARNING: pkg-config not available, cannot check for ao
configure: WARNING: pkg-config not available, cannot check for alsa
configure: WARNING: pkg-config not available, cannot check for samplerate
configure: error: Selected sound system is not available.
```

I'm not sure if this is the right solution for this, so I would appreciate if someone could confirm.

Reply:
Its a bit odd that it seemingly cannot find `pkg-config` as it is listed as a build-dep... Perhaps the reconf is needed to fix a cached previous incorrect setting...